test/UnitTest/v2/test_casesNGSIv2.py

```diff
@@ -33,149 +34,153 @@ def test_getAllSubscription():
+        r=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        resp_content=r.content
+        resInJson= resp_content.decode('utf8').replace("'", '"')
+        resp=json.loads(resInJson)
+        resp=resp['subscribeResponse']
+        sid=resp['subscriptionId']
+        get_url=brokerIp+"/v2/subscription/"
+        url=get_url+sid
+        r=requests.delete(url,headers=headers)
+        assert r.status_code == 200
+
+
 
 #testCase 5
-''' 
+'''
 Test if subscriptionId is not persent in the broker
 '''
 
-def test_deleteSubscriptionId():	
-	delete_url=brokerIp+"/v2/subscription/nil"
-	headers= {'Content-Type': 'application/json'}
-	r=requests.delete(delete_url,headers=headers)
-	assert r.status_code == 200
+def test_deleteSubscriptionId():
+        delete_url=brokerIp+"/v2/subscription/nil"
+        headers= {'Content-Type': 'application/json'}
+        r=requests.delete(delete_url,headers=headers)
+        assert r.status_code == 200
 
 #testCase 6
 '''
   Test with wrong payload
 '''
 
 def test_subscriptionWithWrongPayload():
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	r=requests.post(url,data=json.dumps(v2data.subscriptionWrongPaylaod),headers=headers)
-	assert r.status_code == 500
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        r=requests.post(url,data=json.dumps(v2data.subscriptionWrongPaylaod),headers=headers)
+        assert r.status_code == 500
 
 #testCase 7
 '''
   Testing subscription response with the same entity in ngsiv1 and ngsiv2.
 '''
 
 def test_v1v2SubscriptionForSameEntity():
-	V2url=brokerIp+"/v2/subscriptions"
-	ngsi10url=brokerIp+"/ngsi10/subscribeContext"
-	headers= {'Content-Type': 'application/json'}
-	V2=requests.post(V2url,data=json.dumps(v2data.subscription_data),headers=headers)
-	ngsi10=requests.post(ngsi10url,data=json.dumps(v2data.v1SubData),headers=headers)
-	assert V2.status_code == 201
-	assert ngsi10.status_code == 200
-	
+        V2url=brokerIp+"/v2/subscriptions"
+        ngsi10url=brokerIp+"/ngsi10/subscribeContext"
+        headers= {'Content-Type': 'application/json'}
+        V2=requests.post(V2url,data=json.dumps(v2data.subscription_data),headers=headers)
+        ngsi10=requests.post(ngsi10url,data=json.dumps(v2data.v1SubData),headers=headers)
+        assert V2.status_code == 201
+        assert ngsi10.status_code == 200
+
 #testCase 8
 '''
  Test ngsiv2 subscription
 '''
 
-def test_Subscription():	
-	url=brokerIp+"/v2/subscriptions"
-	headers= {'Content-Type': 'application/json'}
-	response=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
-	assert response.status_code==201
+def test_Subscription():
+        url=brokerIp+"/v2/subscriptions"
+        headers= {'Content-Type': 'application/json'}
+        response=requests.post(url,data=json.dumps(v2data.subscription_data),headers=headers)
+        assert response.status_code==201
 
```

Comment:
For test case #9, #10, #11: Is it correct? Payload appears to be of v1 update and url is /v2/subscriptions/, what is actually being tested here? Assert checks whether the update is successful.